conf/test_cache.config

```diff
@@ -0,0 +1,50 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests using cached / offline test data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/rnaseq -profile test,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name        = 'Test profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+
+    // Input data
+    // params.test_data_base (default) = 'https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq3/'
+    input = "${params.test_data_base}/samplesheet/v3.10/samplesheet_test.csv"
+
+    // Genome references
+    fasta              = "${params.test_data_base}/reference/genome.fasta"
```

Comment:
Yep, the `test_cache.config` will be used in Github Actions so we can clone the branch with the test data (and be able to do the same in offline environments) to run the minimal tests - this is mostly because we had CI tests failing quite frequently when using absolute paths. The `test.config` was reverted to be the same as other pipelines with absolute paths.

Reply:
we need to do the same with methylseq and sarek